### DISCUSSION TITLE
Can I transfer flakelight to nix-community?

### DISCUSSION THREAD

**Discussion by accelbread** [nix-community/infra#1043] [Q&A] [answered]
I have a library for defining flakes: https://github.com/accelbread/flakelight
It was also discussed on Discourse here: https://discourse.nixos.org/t/flakelight-a-new-modular-flake-framework/32395

**Comment by Mic92** [comment, marked as answer]
Invite has been sent. You can join nix-community by visiting https://github.com/nix-community
Afterwards you can transfer and report back. I will than create a team of which you are the admin.

**Reply by accelbread** [reply]
Thanks, transferred: https://github.com/nix-community/flakelight

**Reply by Mic92** [reply]
Done. You can add collaborators now by adding them to the team.
